gatsby/content/blog/2020/05/2020-05-27-room-v6-s2s-r0.1.4-c2s-r0.6.1.mdx

```diff
@@ -0,0 +1,88 @@
+---
+date: '2020-05-27'
+title: 'Room Version 6, Client-Server r0.6.1, and Federation r0.1.4 spec releases'
+categories:
+  - Releases
+author: Travis Ralston
+---
+
+## Room Version 6, Client-Server r0.6.1, and Federation r0.1.4 spec releases
+
+Hey all,
+
+It's been [a little while](https://matrix.org/blog/2019/11/08/client-server-r-0-6-0-and-identity-service-r-0-3-0-releases) since we've done a spec release, so here we are with [Room Version 6](https://matrix.org/docs/spec/rooms/v6), [Client-Server r0.6.1](https://matrix.org/docs/spec/client_server/r0.6.1), and [Federation r0.1.4](https://matrix.org/docs/spec/server_server/r0.1.4).
+
+Room Version 6 (and the associated Federation r0.1.4 release) is largely something for implementations to worry about. It contains new event authorisation rules, changes to the redaction algorithm, and stricter compliance for JSON.
+
+Client-Server r0.6.1 contains a number of clarifications as well as SSO support for authorization, "soft logout" to avoid needlessly destroying e2e history, and new ways to publish aliases within rooms.
```

Comment:
```suggestion
Client-Server r0.6.1 contains a number of clarifications as well as SSO support for authorisation, "soft logout" to avoid needlessly destroying e2e history, and new ways to publish aliases within rooms.
```